cpp/client/async_log_client.cc

```diff
@@ -464,12 +464,13 @@ void AsyncLogClient::InternalAddChain(const CertChain& cert_chain,
   jsend.Add("chain", jchain);
 
   UrlFetcher::Request req(GetURL(pre_cert ? "add-pre-chain" : "add-chain"));
+  req.verb = UrlFetcher::Verb::POST;
```

Comment:
Yeah, I take your point, just seems like the verb might be one of the easier ones to forget to set resulting in some head scratching.  But it's your call.